grafana/dashboards/cmode/cdot.json

```diff
@@ -2194,14 +2194,14 @@
       },
       {
         "current": {},
-        "definition": "query_result(topk($TopResources, (100 * volume_size_used{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\",volume=~\"$Volume\"} / volume_size_total{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\",volume=~\"$Volume\"})))",
+        "definition": "query_result(topk($TopResources, (100 * volume_size_used{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\",volume=~\"$Volume\"} / volume_size_total{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\",volume=~\"$Volume\"}) * on (cluster, svm, volume) volume_labels{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\",volume!=\"vol0\"}))",
```

Comment:
Agreed that, I have tried to check vserver_root_volume and node_root_volume. The vol0s are node_root_volumes only. 

<img width="1773" alt="image" src="https://user-images.githubusercontent.com/83282894/204841174-761445c2-eaa9-4821-8a5d-26de7cb803f5.png">

It would be better to ask customer regarding the vserver_root as well.